### ISSUE TITLE
Why images are all black after using 'random_adjust_brightness'?

### ISSUE THREAD

**Issue by liangxiao05** [tensorflow/models#3834]
I meet a very strange problem:
I display the training images on tensorboard by adding this behind 
[https://github.com/tensorflow/models/edit/master/research/object_detection/trainer.py#L167](url)
```
#show images
with tf.name_scope('images'):
  tf.summary.image('images',images,max_outputs=24)**
```
Here my data augmentation config is this，and the images shown are correct.
```
  data_augmentation_options {
    random_horizontal_flip {
    }
  }
  data_augmentation_options {
    ssd_random_crop {
    }
  }
```
But if I add a new function like below,then the images shown are all black 
```
  data_augmentation_options {
    random_horizontal_flip {
    }
  }
  data_augmentation_options {
    ssd_random_crop {
    }
  }
  data_augmentation_options {
    random_adjust_brightness {
    max_delta:0.2
    }
  }
```
I don't know why ,the pixels values inputs are both between [0,1]  in 'random_adjust_brightness' and  'ssd_random_crop', so if the images shown are correct before there is no reason to be uncorrect now, anyone puzzled by this ? needs help ~~~!!!!!

**Comment by bitfort** [comment]
I've added @jch1 to take a look and help :)

**Comment by liangxiao05** [comment]
@bitfort ，thank you
I found that if added 'random_adjust_brightness ',the loss of the ssd model can't decrease.
After trainging 48W steps,the loss has little change compared to 1k steps , however if removed the  'random_adjust_brightness ' ,the average loss at 40W+ step is below 3.
''
train_config: {
  batch_size: 24
  optimizer {
    adam_optimizer: {
      learning_rate: {
        exponential_decay_learning_rate {
          initial_learning_rate: 0.002
          decay_steps: 40000
          decay_factor: 0.95
        }
      }
    }
  }

INFO:tensorflow:global step 488997: loss = 6.7185 (0.685 sec/step)
INFO:tensorflow:global step 488998: loss = 8.5463 (0.598 sec/step)
INFO:tensorflow:global step 488999: loss = 8.3720 (0.618 sec/step)
INFO:tensorflow:global step 489000: loss = 4.3897 (0.631 sec/step)
INFO:tensorflow:global step 489001: loss = 8.6147 (0.622 sec/step)
INFO:tensorflow:global step 489002: loss = 7.7338 (0.650 sec/step)
INFO:tensorflow:global step 489003: loss = 6.8271 (0.739 sec/step)
INFO:tensorflow:global step 489004: loss = 7.2645 (0.651 sec/step)
''

**Comment by mbenami** [comment]
Hi @liangxiao05 
there is a bug in `random_adjust_brightnessz` 
duplicate of  #3637 , #2066 
fix that help me: #2146 

don't know why this PR is still hasn't merge
